src/plugins/vis_builder/public/application/utils/state_management/redux_persistence.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { VisBuilderServices } from '../../../types';
+import { getPreloadedState } from './preload';
+import { RootState } from './store';
+
+export const loadReduxState = async (services: VisBuilderServices) => {
+  try {
+    const serializedState = services.osdUrlStateStorage.get<RootState>('_a');
+    if (serializedState === null) {
+      return await getPreloadedState(services);
+    }
+    return serializedState;
+  } catch (err) {
+    return await getPreloadedState(services);
+  }
```

Comment:
nit: 
```suggestion
  try {
    const serializedState = services.osdUrlStateStorage.get<RootState>('_a');
    if (serializedState !== null) return serializedState;
  } catch (err) {
    console.error(err); // this may need a ts-ignore statement since console statements are not allowed by default in OSD
  }
  
  return await getPreloadedState(services);
```

Reply:
Made the suggested changes, and add ts-ignore statements.